invoices/invoice_expiry_watcher.go

```diff
@@ -353,6 +353,10 @@ func (ew *InvoiceExpiryWatcher) expireInvoice(hash lntypes.Hash, force bool) {
 
 	case channeldb.ErrInvoiceAlreadySettled:
 
+	case channeldb.ErrInvoiceNotFound:
+		// It's possible that the user already canceled the invoice
+		// which then will be deleted by the time we'll try to cancel.
```

Comment:
nit: mention the auto garbage collection here as well since that's the cause of this?

Reply:
done